scalding-core/src/main/scala/com/twitter/scalding/RichPipe.scala

```diff
@@ -53,6 +54,38 @@ object RichPipe extends java.io.Serializable {
     }
     p
   }
+
+  // A pipe can have more than one description when merged together, so we store them delimited with 254.toChar.
+  private def encodePipeDescriptions(descriptions: Seq[String]): String = {
+    descriptions.map(_.replace(254.toChar, ' ')).filter(_.nonEmpty).mkString(254.toChar.toString)
```

Comment:
can we factor `254.toChar` into a private val? Also, can you comment on the choice here? What impact does this have for UTF-8 descriptions? A guess since means that the 254 code point:  http://www.fileformat.info/info/unicode/char/fe/index.htm can't appear in descriptions. Why not `1.toChar` which is often used as a delimiter on HDFS?